frc42_dispatch/generate_hashes.py

```diff
@@ -0,0 +1,27 @@
+from hashlib import blake2b
+
+# See https://github.com/filecoin-project/FIPs/blob/master/FRCs/frc-0042.md#method-number-computation
+def method_number(name):
+    name = '1|' + name
+    hash = blake2b(name.encode('ascii'))
```

Comment:
Pass `digest_size=64` explicitly just to be concrete (but it is the default too).